### ISSUE TITLE
Improve visual output when `viash ns build` fails.

### ISSUE THREAD

**Issue by DriesSchaumont** [viash-io/viash#444]
When running `viash ns build` and the building of a component fails, a user might get
```
Not all configs built successfully # in yellow
  2/149 configs were disabled. # in yellow
  146/149 configs built successfully # in green
```

It might be useful to add the number of failed components in red, to visually indicate that something went wrong (right now, the colors dont chang when the build fais)

```
Not all configs built successfully # in yellow
  2/149 configs were disabled. # in yellow
  146/149 configs built successfully # in green
  1/149 configs failed # in red
```

**Comment by DriesSchaumont** [comment]
To reproduce:

```shell
# go to openpipelines repo
git fetch origin
git checkout 0.8.0
```

Replace the `platforms` in the config file for multi_star_to_h5mu (`./src/mapping/multi_star_to_h5mu`) with:

```yaml
platforms:
  - type: docker
    image: python:3.10
    setup:
      - type: python
        packages:
          - anndata~=0.8.0
          - mudata~=0.2.0
          - filiberke # This is clearly wrong
```

And run:

```
viash ns build -q multi_star_to_h5mu --parallel --setup cachedbuild
```

```
viash ns build -q multi_star_to_h5mu --parallel --setup cachedbuild
Exporting multi_star_to_h5mu (mapping) =docker=> /home/di/code/openpipeline/target/docker/mapping/multi_star_to_h5mu
[notice] Building container 'ghcr.io/openpipelines-bio/mapping_multi_star_to_h5mu:dev' with Dockerfile
[error] Error occurred while building container 'ghcr.io/openpipelines-bio/mapping_multi_star_to_h5mu:dev'
[error] Transcript: --------------------------------
#1 [internal] load .dockerignore
#1 transferring context: 2B done
#1 DONE 0.0s

#2 [internal] load build definition from Dockerfile
#2 transferring dockerfile: 649B done
#2 DONE 0.0s

#3 [internal] load metadata for docker.io/library/python:3.10
#3 DONE 0.4s

#4 [1/2] FROM docker.io/library/python:3.10@sha256:0da028d058c56277dc3e5f197bcdc53ffe2dd5c6cd947b48747c8a8d24911be4
#4 CACHED

#5 [2/2] RUN pip install --upgrade pip &&   pip install --upgrade --no-cache-dir "anndata~=0.8.0" "mudata~=0.2.0" "filiberke"
#5 0.892 Requirement already satisfied: pip in /usr/local/lib/python3.10/site-packages (23.0.1)
#5 0.974 Collecting pip
#5 1.084   Downloading pip-23.1.2-py3-none-any.whl (2.1 MB)
#5 1.171      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.1/2.1 MB 24.0 MB/s eta 0:00:00
#5 1.203 Installing collected packages: pip
#5 1.203   Attempting uninstall: pip
#5 1.203     Found existing installation: pip 23.0.1
#5 1.256     Uninstalling pip-23.0.1:
#5 1.300       Successfully uninstalled pip-23.0.1
#5 1.809 Successfully installed pip-23.1.2
#5 1.809 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#5 2.151 Collecting anndata~=0.8.0
#5 2.204   Downloading anndata-0.8.0-py3-none-any.whl (96 kB)
#5 2.220      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 96.1/96.1 kB 5.7 MB/s eta 0:00:00
#5 2.240 Collecting mudata~=0.2.0
#5 2.248   Downloading mudata-0.2.3-py3-none-any.whl (24 kB)
#5 2.383 ERROR: Could not find a version that satisfies the requirement filiberke (from versions: none)
#5 2.383 ERROR: No matching distribution found for filiberke
#5 ERROR: process "/bin/sh -c pip install --upgrade pip &&   pip install --upgrade --no-cache-dir \"anndata~=0.8.0\" \"mudata~=0.2.0\" \"filiberke\"" did not complete successfully: exit code: 1
------
 > [2/2] RUN pip install --upgrade pip &&   pip install --upgrade --no-cache-dir "anndata~=0.8.0" "mudata~=0.2.0" "filiberke":
#5 1.203     Found existing installation: pip 23.0.1
#5 1.256     Uninstalling pip-23.0.1:
#5 1.300       Successfully uninstalled pip-23.0.1
#5 1.809 Successfully installed pip-23.1.2
#5 1.809 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv

#5 2.240 Collecting mudata~=0.2.0
#5 2.248   Downloading mudata-0.2.3-py3-none-any.whl (24 kB)
#5 2.383 ERROR: Could not find a version that satisfies the requirement filiberke (from versions: none)
#5 2.383 ERROR: No matching distribution found for filiberke
------
Dockerfile:3
--------------------
   2 |     
   3 | >>> RUN pip install --upgrade pip && \
   4 | >>>   pip install --upgrade --no-cache-dir "anndata~=0.8.0" "mudata~=0.2.0" "filiberke"
   5 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c pip install --upgrade pip &&   pip install --upgrade --no-cache-dir \"anndata~=0.8.0\" \"mudata~=0.2.0\" \"filiberke\"" did not complete successfully: exit code: 1
[error] End of transcript --------------------------
Exporting multi_star_to_h5mu (mapping) =nextflow=> /home/di/code/openpipeline/target/nextflow/mapping/multi_star_to_h5mu
Not all configs built successfully
  71/73 configs were disabled
  1/73 configs built successfully
```

**Comment by Grifs** [comment]
Fixed in Viash 0.7.5
